css/style.css

```diff
@@ -1,19 +1,170 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  margin: 0;
 }
 
 /**
  * Add your custom styles below
  * 
  * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
+ * - Be organized, use comments and separate your styles into meaningful chunks
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/*header */
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 3rem;
+  margin-bottom: 1rem;
+  padding-top: 1rem;
+}
+
+.header-logo {
+  display: flex;
+  max-height: 50px;
+  width: 100px;
+  margin-left: 15px;
+}
+
+/* Navigation */
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin-right: 80px;
+}
+
+.nav-item {
+  padding: 0.5rem 1rem;
+}
+
+#nav-bold {
+  color: rgb(116, 113, 113);
+  font-weight: 500;
+}
+.nav-link {
+  color: rgb(160, 155, 155);
+  text-decoration: none;
+  font-size: 1.4rem;
+}
+
+.nav-link:hover {
+  color: rgb(255, 102, 0);
+  cursor: pointer;
+}
+
+/* main */
+
+.content p {
+  text-align: center;
+  font-size: 3rem;
+  font-weight: 100;
+}
+
+.center {
+  background-image: url("../img/first-background.jpg");
+  height: 40vw;
+  background-repeat: no-repeat;
+  padding-top: 15.3rem;
+  background-size: 117%;
+  background-position-x: center;
+  padding-right: 5%;
+}
```

Comment:
Is there a way that we could change this to use flex-box to position the text in the center of the image instead of padding? doing it this way we have had to add padding and text-align elsewhere so it would probably give a neater solution with less lines of css :)